app/jobs/stockit_sync_orders_package_job.rb

```diff
@@ -0,0 +1,22 @@
+class StockitSyncOrdersPackageJob < ActiveJob::Base
+  queue_as :stockit_sync_orders_package_updates
+
+  def perform(package_id, orders_package, operation)
+    package = Package.find_by(id: package_id)
+
+    if package
+      if(operation == "create")
```

Comment:
A case / switch statement might be nice here to tidy the code up. (non-essential)